pkg/clusteragent/orchestrator/controller.go

```diff
@@ -151,10 +140,8 @@ func newController(ctx ControllerContext) (*Controller, error) {
 		clusterName:             ctx.ClusterName,
 		clusterID:               clusterID,
 		orchestratorConfig:      orchestratorCfg,
-		simpleDataScrubber:      orchestrator.NewDefaultDataScrubber(),
 		forwarder:               forwarder.NewDefaultForwarder(podForwarderOpts),
 		isLeaderFunc:            ctx.IsLeaderFunc,
-		isScrubbingEnabled:      config.Datadog.GetBool("orchestrator_explorer.container_scrubbing.enabled"),
 		extraTags:               config.Datadog.GetStringSlice("orchestrator_explorer.extra_tags"),
```

Comment:
do we want to move the extraTags also under `orchestratorCfg`?
Would be one less parameter in the other functions 🤔